.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: ruby
+rvm:
+  # SLES15 sp0
+  - 2.4.1
+
+  # Future versions
+  - ruby-head
+script:
+  - 'bundle exec rspec spec'
+  - 'bundle exec rubocop'
```

Comment:
Does Travis stop and report after it hit the first error? If yes, I would move rubocop one line up (before rspec) as rubocop usually fails faster than a large number of rspec tests.
Could save us some time.